evmdis/main.go

```diff
@@ -5,29 +5,96 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-
-	"github.com/Arachnid/evmdis"
+	"log"
+	"strconv"
+	"flag"
+	"github.com/soad003/evmdis"
```

Comment:
ok, i have read that there are no relative imports in golang. Your repo with another remote would be an option. But i think i leave it like that for now and change the ref before i push something...